miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -365,6 +369,41 @@ def map(self):
         # returns ['retry'] without internet
         return self.send("get_map_v1")
 
+    @command()
+    def get_multi_maps(self) -> MultiMapList:
+        """Return list of multi maps."""
+        if self._multi_maps is not None:
+            return self._multi_maps
+
+        self._multi_maps = MultiMapList(self.send("get_multi_maps_list")[0])
+        return self._multi_maps
+
+    @command()
+    def multi_map_enum(self) -> Optional[enum.Enum]:
+        """Enum of the available map names."""
```

Comment:
I thought I mentioned about this, but if this is just for internal use, removing `@command()` and prefixing the name with `_` should make it clear for anyone using the API.

Reply:
I renamed the function to _map_enum, the internal variable to _map_enum_cache.
I also removed the @command()